### DISCUSSION TITLE
Access Key and Access Secret behaviour

### DISCUSSION THREAD

**Discussion by ajguidet** [casdoor/casdoor#3079] [Q&A]
I want to use the `Access Key` and `Access Secret` in my application, reading the documentation 
https://casdoor.org/docs/basic/public-api/#3-by-access-key-and-access-secret and looking in this https://github.com/casdoor/casdoor/pull/1971. The problem is that I can only get it to work for the built-in organisation, not for other manually created organisations. Is there anything additional that needs to be configured?

**Comment by Neerajgowda2611** [comment]
No answer did you find a solution for this??
i hosted casdoor all in one image using docker but i cant see the generate access key / access secret options in the user settings or  ui 
any kind of guidance will be appreciated
